File: project/hyeok-onboarding/src/main/java/com/INNER_CIRCLE_ICD4/innerCircle/survey/request/repository/ResponseRepository.java
package com.INNER_CIRCLE_ICD4.innerCircle.survey.request.repository;

import com.INNER_CIRCLE_ICD4.innerCircle.survey.request.domain.Response;
import org.springframework.data.jpa.repository.JpaRepository;

public interface ResponseRepository extends JpaRepository<Response, Long> {
}

File: project/soojinhan-onboarding/src/main/kotlin/onboardingproject/project/repository/ResponseRepository.kt
package onboardingproject.project.repository

import onboardingproject.project.domain.Response
import org.springframework.data.jpa.repository.JpaRepository

/**
 * packageName : onboardingproject.project.repository
 * fileName    : ResponseRepository
 * author      : hsj
 * date        : 2025. 6. 14.
 * description :
 */
interface ResponseRepository : JpaRepository<Response, String>